README.md

```diff
@@ -129,4 +130,4 @@ setting, updating or using another waypoint.
 ![commands-wp-undo](images/commands-wp-undo.png)
 
 Waypoints can be cleared by executing `/wp clear`. Be careful, this action will remove all player waypoints and cannot
-be rollback.
\ No newline at end of file
+be rollback.
```

Comment:
No need LF here

Reply:
That is actually done automatic can't even change it back.